src/Services/ServicesClass/DemoService.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace Cms\Services\ServicesClass;
+
+class DemoService
+{
+  public function helloWorld()
+  {
+    return "Hello World!\n";
+  }
+
+  public function currentDate(){
+    $date = date('d/m/Y h:i:s');
+    return $date;
+  }
+}
```

Comment:
No new line at EOF - add new line

Reply:
@Libbna Updated the PR with requested changes. Please review it